reports/California.md

```diff
@@ -182,6 +200,20 @@ id: ca-longbeach-2
 
 ## Los Angeles
 
+### Police use tear gas and riot rounds against peaceful protestors | May 29th
```

Comment:
Small point, but from the video and post it isn't clear the protestors are peaceful. It would be good to have another source referencing peaceful demonstrations prior to the police's actions.